src/cmd/flags.go

```diff
@@ -123,6 +125,7 @@ func bindFlags() {
 
 	flag.BoolVar(&linter.CheckAutoGenerated, `check-auto-generated`, false, "whether to lint auto-generated PHP file")
 	flag.BoolVar(&linter.Debug, "debug", false, "Enable debug output")
+	flag.StringVar(&debugParseDuration, "debug-parse-duration", "0ms", "Print files that took longer than the specified time to analyse")
```

Comment:
It looks like it can be `flag.DurationVar(&linter.DebugParseDuration, )`, unless I'm missing something.

Reply:
It can if it exists... For some reason I didn't think it does.

Reply:
Fixed